HDS-5192 Adds support for `aria-describedby` in `Reveal` component (#3084)
hashicorp/design-system: website/docs/components/reveal/partials/code/component-api.md

  <C.Property @name="textWhenOpen" @type="string">
    Plain text which displays on the toggle button while the content is displayed.
  </C.Property>

>   <C.Property @name="ariaDescribedBy" @type="string">

Reviewer: [todo] usually the documentation is updated in a separate PR, so that it can be merged only when the changes are released; in this case they're minimal, so I'll leave it to you to decide if you want to split the PR or not

>   <C.Property @name="ariaDescribedBy" @type="string">
>     If given the value of an element's ID, will be added as the toggle's `aria-description` attribute value.
>   </C.Property>

Reviewer 2: ```suggestion
  <C.Property @name="ariaDescribedBy" @type="string">
    Accepts a string. The ariaDescribedBy value is applied to the toggle's `aria-description` attribute value.
  </C.Property>
```
